pkgs/top-level/make-tarball.nix

```diff
@@ -64,7 +64,7 @@ pkgs.releaseTools.sourceTarball {
     echo "generating packages.json"
     mkdir -p $out/nix-support
     echo -n '{"version":2,"packages":' > tmp
-    nix-env -f . -I nixpkgs=$src -qa --meta --json --show-trace --arg config 'import ${./packages-config.nix}' "''${opts[@]}" >> tmp
+    nix-env -f . -I nixpkgs=$src -qa --meta --json --out-path --show-trace --arg config 'import ${./packages-config.nix}' "''${opts[@]}" >> tmp
```

Comment:
Sad to see `nix-env` is still alive :). Looks good to me overall, but I don't feel confident to merge this, as I don't understand what it does. Did you find docs for this option?